.storybook/storybook-data.js

```diff
@@ -580,19 +580,6 @@ module.exports = [
 			},
 		},
 	},
-	{
-		id: 'wordpress--wordpress-activation',
```

Comment:
❤️ Thanks so much for this fix! I wouldn't have thought to look in that folder to fix the tests! 😅